kong/plugins/oidc/handler.lua

```diff
@@ -29,27 +29,52 @@ function CustomHandler:access(config)
 
     session.configure(config)
 
-    local res, err = require("resty.openidc").authenticate(oidcConfig)
+    if tryIntrospect(oidcConfig) then
 
-    if err then
-      if config.recovery_page_path then
-        ngx.log(ngx.DEBUG, "Entering recovery page: " .. config.recovery_page_path)
-        return ngx.redirect(config.recovery_page_path)
+      ngx.log(ngx.DEBUG, "In plugin CustomHandler:proceeding with two legged authentication, requested path: " .. ngx.var.request_uri)
```

Comment:
This log is kinda misleading. Sending a token doesn't mean the client used a two legged flow. It's out of scope for this plugin. It should only check, if the token it gets (no matter, if it's a access token, AAT, RPT) was created by the Authorization Server.
Also the `CustomHandler` is misleading, but IMO this is out of scope for this MR.

Reply:
Fixed -> now it says that valid token can go through